tests/scripts/github-action-helper.sh

```diff
@@ -805,6 +805,18 @@ function delete_cluster() {
   remove_cluster_prerequisites
 }
 
+function checkIfTheKeyExists() {
```

Comment:
You might consider allowing this function to take a list of keys to check for existence. This could make it easier to read/write tests after my request for more detailed tenant side-effect checks. I'll let you decide whether it actually helps with your test writing or not, so please consider this review comment optional.

Something like this maybe

```sh
toolbox="$( ...etc... )"
for key in "${@}"; do
  if ...etc...; then
    echo "key '$key' exists"
  else
    echo "key '$key' doesn't exist"
    exit 1
  fi
done
```